### DISCUSSION TITLE
How old does the account have to be to apply for the GitHub Education project?

### DISCUSSION THREAD

**Discussion by 5DMax** [community/community#63928] [GitHub Education] [answered]
### Hi everyone,

Since I've recently created my GitHub account and have already submitted the necessary educational documents, I've encountered an issue regarding the account's relatively short existence. I'm curious to know the specific timeframe required for an account's eligibility to submit documents, Thank you.

**Comment by oftharsh** [comment]
If you like it please mark it as the answer.

Hello,

Congratulations on creating your GitHub account and taking the steps to verify your student status! Typically, GitHub's Education team processes and verifies student accounts within a few days to a couple of weeks after you've submitted your documents. However, the exact timeframe can vary based on factors like the volume of verification requests and the current workload of the GitHub Education team.

If you've already submitted the necessary educational documents, I recommend keeping an eye on your email inbox associated with your GitHub account. GitHub will usually send you an email notification once your account has been verified, and you'll gain access to the benefits of the GitHub Student Developer Pack.

In the meantime, if you have any urgent inquiries or if you believe there might be a specific issue with the verification process, you can reach out to GitHub Support or check their official documentation for more information about the verification process and expected timeframes.

Remember that the verification process is in place to ensure that eligible students can access the benefits provided by the GitHub Student Developer Pack, so while waiting, you can continue exploring GitHub and its features.

**Reply by 5DMax** [reply]
I mean the age of the account. Since I just joined 3 days ago and after submitting all my paperwork. It has a warning message saying "Your GitHub account was too recently created. Please apply again in a few days."

**Reply by oftharsh** [reply]
GitHub might have certain restrictions on newly created accounts to prevent misuse or spam. This is a common practice on many online platforms to ensure the safety and integrity of the community. The specific duration of the waiting period can vary, but it's often a few days to a week.

If you've received a message saying that your GitHub account was too recently created and to apply again in a few days, it means that you need to wait until the specified waiting period is over before you can proceed with submitting your educational documents or accessing certain features.

During this waiting period, you can take the time to explore GitHub, learn about its features, and prepare any necessary documents that you'll need to submit once the waiting period is over. If you're still unable to proceed after the waiting period has passed, you might want to reach out to GitHub's support or help center for further assistance.

Remember that the waiting period is in place to help maintain the quality and security of the GitHub community, and it's a standard practice for many online platforms.

**Reply by 5DMax** [reply]
Thank you very much.

**Comment by nirav-gajera** [comment, marked as answer]
According to the GitHub documentation, a new GitHub account is eligible to submit documents after 30 days of creation.

Here are the specific timeframes for a GitHub account's eligibility to submit documents:

* **30 days:** New accounts are eligible to submit documents after 30 days of creation.
* **1 year:** Accounts that have been inactive for more than 1 year are not eligible to submit documents.
* **Permanently:** Accounts that have been banned are permanently ineligible to submit documents.

If you have recently created your GitHub account and have already submitted the necessary educational documents, but you are still getting an error message, you can contact [GitHub support](https://support.github.com/) for help.

**Reply by 5DMax** [reply]
Thank you, and I have to wait for the account to be more than 30 days old to submit the documents, is that correct?

**Reply by nirav-gajera** [reply]
yes , sometimes its done before 30 days in some case

**Reply by 5DMax** [reply]
Thank you very much.
